internal/controller/drpolicy_peerclass.go

```diff
@@ -136,14 +136,36 @@ func updatePeerClassStatus(u *drpolicyUpdater, syncPeers, asyncPeers []peerInfo)
 	return u.statusUpdate()
 }
 
+// vscProvisionerMatchesSC inspects VolumeSnapshotClass in the passed in classLists at the specified index, and
+// matches it to the provisioner in the StorageClass, and returns true if matched
+func vscProvisionerMatchesSC(scName string, cl classLists, vscIdx int) bool {
```

Comment:
This function, along with `vrcProvisionerMatchesSC`, caused some confusion as I initially thought there was an incorrect comparison, not realizing there were two functions performing the same task. I suggest renaming the function to `provisionerMatchesSC` and updating the parameters to:  
`(scName string, sClasses []*storagev1.StorageClass, driver string)`

Reply:
Nice change, updated. (but squashed both commits into one as a result :/ )